avm/res/network/private-endpoint/main.bicep

```diff
@@ -287,12 +255,12 @@ type privateLinkServiceConnectionsType = {
     @description('Optional. A message passed to the owner of the remote resource with this connection request. Restricted to 140 chars.')
     requestMessage: string?
   }
-}[]?
+}
 
 type customDnsConfigType = {
   @description('Required. Fqdn that resolves to private endpoint IP address.')
   fqdn: string
 
   @description('Required. A list of private IP addresses of the private endpoint.')
   ipAddresses: string[]
-}[]?
+}
```

Comment:
Should this top level type also be exported?

Reply:
Well ... yes

Reply:
Updated. Thanks